### PR TITLE
Feature/improve poor galaxy score

### PR DESCRIPTION
**Pull request by jaredledvina**
Working branch for closing https://github.com/sensu/sensu-ansible/issues/217 opening a PR to kick of Travis so I don't need to drain my battery running all these tests.

### CODE REVIEW

**Comment by jaredledvina** [comment]
There, that should get most of them.

**Comment by michaelpporter** [comment]
#219 addresses the last of the lint issues

File: tasks/CentOS/rabbit.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
 # HACK: https://github.com/ansible/ansible/issues/20711#issuecomment-306260869
 # Can be removed once we're running w/ a version of Ansible that has https://github.com/ansible/ansible/pull/35989
-- name: Make yum cache to import GPG keys # noqa 503 - Order really matters here and flush handlers is too much for this
+- name: Make yum cache to import GPG keys0  # noqa 503 - Order really matters here and flush handlers is too much for this
```
Comment (michaelpporter) [inline tasks/CentOS/rabbit.yml]: Oops, I entered an extra 0 in the name.

Comment (jaredledvina) [reply tasks/CentOS/rabbit.yml]: Yep, just dropped it, no worries!

File: .yamllint [hunk 2]
```diff
@@ -1,11 +1,21 @@
+---
+# https://github.com/ansible/galaxy/blob/devel/galaxy/importer/linters/yamllint.yaml
```
Comment (michaelpporter) [inline .yamllint:2]: Much better solution than me adding spaces. I learned this today, thank you.

Comment (jaredledvina) [reply .yamllint:2]: ha, nice! Yeah, I noticed this from https://galaxy.ansible.com/docs/contributing/content_scoring.html#linters-used actually. Seems sane to me.

**Comment by jaredledvina** [comment]
Wooo and Travis passed! I think we're good to ship this!

**Pull request merged**